test/document_stores/test_base.py

```diff
@@ -318,7 +318,7 @@ def test_duplicate_documents_skip(self, ds, documents):
 
         ds.write_documents(updated_docs, duplicate_documents="skip")
         for d in ds.get_all_documents():
-            assert d.meta["name"] != "Updated"
+            assert d.meta.get("name") != "Updated"
```

Comment:
I understand that this change ensure that no KeyError is raised if `"name"` is not part of `d.meta` and instead `Null` is returned. My question is if that was an issue before? The documents all have meta data with a name set, no? 
There is nothing wrong about this change. I just would like to understand it a bit better. 🙂